ipv8/community.py

```diff
@@ -643,7 +621,8 @@ def get_new_introduction(self, from_peer: Peer | None = None) -> None:
             available = self.get_peers()
             if available:
                 # With a small chance, try to remedy any disconnected network phenomena.
-                if self._bootstrappers_initialized and random() < 0.05:
+                bootstrappers_ready = all(b.initialized for b in self.bootstrappers)
```

Comment:
I don't understand the choice of `all` over `any` here. It seems to me that the latter is preferable.

Reply:
I used `all` to remain as close as possible to the old code. Before `_bootstrappers_initialized` would be set to `True` if all bootstrappers were initializing or initialized. Hence my choice to do the same here. Having said that, it probably doesn't matter so I'll change it.